frontend/src/util/modalStyles.ts

```diff
@@ -0,0 +1,22 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const modalStyles = makeStyles((theme) => ({
+  modal: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  paper: {
+    position: 'absolute',
+    width: '80%',
+    maxWidth: 400,
+    backgroundColor: theme.palette.background.paper,
+    boxShadow: theme.shadows[5],
+    padding: theme.spacing(2, 4, 3),
+  },
+  modalSwitchText: {
+    paddingTop: '10px',
+  },
+}));
+
+export default modalStyles;
```

Comment:
Are we extracting our styles now? 

And this should probably not be a 'util'...

Reply:
where should we put it then?

Reply:
Styles folder (if it is remaining extracted)

Reply:
that sounds like a good idea